src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
```

Comment:
The class name 'ColorSupplier' is good, as it refers to a noun and the class is meant to supply colors. However, the class should be public to be accessible from other packages.